android-app/app/src/androidTest/java/de/dhbw/ka/se/fibo/TestMatchers.java

```diff
@@ -1,13 +1,35 @@
 package de.dhbw.ka.se.fibo;
 
 import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
 
 import com.google.android.material.textfield.TextInputLayout;
 
+import org.hamcrest.Description;
 import org.hamcrest.Matcher;
 import org.hamcrest.TypeSafeMatcher;
 
 public class TestMatchers {
+
+    public static Matcher<View> childAtPosition(
+            final Matcher<View> parentMatcher, final int position) {
+
+        return new TypeSafeMatcher<View>() {
```

Comment:
```suggestion
        return new TypeSafeMatcher<>() {
```
Explicit type argument View can be replaced with <>